foundations/installations/installations.md

```diff
@@ -131,14 +131,14 @@ While your VM is running, do the following steps:
   17. Reboot your VM (which you can do by typing `reboot` and hitting enter in a terminal).
   18. You can now maximize the VM window, create additional displays, and use many other useful features. These options are available on the VM toolbar under **View** and **Device**.
 
-  **NOTE**:
+  #### NOTE:
```

Comment:
While checking the Markdown preview, I noticed this wasn't rendering correctly. Removing the spaces seems to fix it.

```suggestion
#### NOTE:
```

Reply:
Should I submit a different Pull Request with this change? I'm happy to knock that out right now.